src/utils/types.ts

```diff
@@ -8,7 +8,8 @@ export interface StaticValue {
 
 export interface ConditionalValue {
   test: string;
-  then: any;
+  // then is reserved for thenables
```

Comment:
I think you can remove the comment. Putting this comment in the PR description should be sufficient.